context/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id "be.insaneprogramming.gradle.animalsniffer" version "1.4.0"
+}
+
+description = 'gRPC: Context'
+
+dependencies {
+    compile libraries.guava,
```

Comment:
@adriancole, I didn't say anything during review, but I was already thinking we'd do a follow-up to remove the dep :). Now works, too.